Source/Dafny/DafnyPipeline.csproj

```diff
@@ -23,6 +23,7 @@
   <ItemGroup>
       <PackageReference Include="JetBrains.Annotations" Version="2021.1.0" />
       <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.7.0" />
+      <PackageReference Include="Moq" Version="4.16.1" />
```

Comment:
What is this used for? I wouldn't expect a `Moq` reference is something other than a testing package.

Reply:
Thank you, you are absolutely right - I think Rider added this line automatically when I experimented with `Moq`. I have removed the reference.